app/src/main/java/net/opendasharchive/openarchive/util/Prefs.kt

```diff
@@ -124,4 +125,14 @@ object Prefs{
 
             prefs?.edit()?.putString(PROOFMODE_ENCRYPTED_PASSPHRASE, passphrase)?.apply()
         }
+
+    var theme: String
+        get() = prefs?.getString(PREF_THEME, null) ?: "system"
+        set(value) {
+            var v: String = value
+            if (!arrayOf("light", "dark").contains(value)) {
```

Comment:
Shouldn't we use an enum here instead of magic strings?

Reply:
fixed, I'm loading these string constants from `resources` now